src/global-messages/GlobalMessagesContext.tsx

```diff
@@ -101,7 +101,12 @@ const GlobalMessagesContextProvider: React.FC = ({children}) => {
   };
 
   const findGlobalMessages = useCallback(
-    (context: GlobalMessageContextType) => {
+    (context?: GlobalMessageContextType) => {
+      if (!context) {
+        return globalMessages.filter((a) =>
+          a.context.some((cont) => cont.includes('app')),
```

Comment:
I believe the check if it includes 'app' is not necessary, as in the Firestore subscription we have this as part of the query:
```ts
.where('context', 'array-contains-any', [
  'app-ticketing',
  'app-departures',
  'app-assistant',
])
```